### PR TITLE
[WEB-5890]: add new message (ProductVoucherAssign)

### PR DESCRIPTION
**Pull request by aosmakserato**

### CODE REVIEW

File: src/Message/ProductVoucherAssign.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Serato\UserProfileSdk\Message;
+
+use DateTime;
```
Comment (aosmakserato) [inline src/Message/ProductVoucherAssign.php:4]: drop this line

File: src/Message/ProductVoucherAssign.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    public function setVoucherId(string $voucherId): self
+    {
+        $this->setParam(self::PARAM_VOUCHER_ID, $voucherId);
+        return $this;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getVoucherId(): ?string
```
Comment (mshahidi) [inline src/Message/ProductVoucherAssign.php:42]: Can a voucherId be `null`?

Comment (aosmakserato) [reply src/Message/ProductVoucherAssign.php:42]: It can if you don't set it.

Comment (aosmakserato) [reply src/Message/ProductVoucherAssign.php:42]: Have a look at `testNullableGetters` method in tests

Comment (mshahidi) [reply src/Message/ProductVoucherAssign.php:42]: Sorry the answer is yes. Ignore this.

**Review by GuillaumeBrousse — CHANGES_REQUESTED** [review]
We don't want to merge this into master right now

**Review by mshahidi — APPROVED** [review]
Looks good @aosmakserato 
You can merge it and create a release.

**Comment by GuillaumeBrousse** [comment]
> Looks good @aosmakserato
> You can merge it and create a release.

Shouldn't we wait for my part to be done and make only one release ? no point of making 2 in a row

**Comment by mshahidi** [comment]
> > Looks good @aosmakserato
> > You can merge it and create a release.
> 
> Shouldn't we wait for my part to be done and make only one release ? no point of making 2 in a row

I don't think there is any SDK changes for the part you are working. It's purely SUP App.

**Comment by GuillaumeBrousse** [comment]
> > > Looks good @aosmakserato
> > > You can merge it and create a release.
> > 
> > 
> > Shouldn't we wait for my part to be done and make only one release ? no point of making 2 in a row
> 
> I don't think there is any SDK changes for the part you are working. It's purely SUP App.

oh yes right, thought it was `sup app`

**Review by GuillaumeBrousse — APPROVED** [review]

**Pull request merged**